src/main/telemetry/srxl.c

```diff
@@ -161,8 +166,20 @@ typedef struct
 } STRU_TELE_FP_MAH;
 */
 
-void srxlFrameFlightPackCurrent(sbuf_t *dst)
+#define FP_MAH_KEEPALIVE_TIME_OUT 2000 // mS
+
+bool srxlFrameFlightPackCurrent(sbuf_t *dst, timeUs_t currentTimeUs)
 {
+  uint16_t amps = getAmperage() / 10;
+  uint16_t caps = getMAhDrawn();
+  static uint16_t sentAmps;
+  static uint16_t sentCaps;
+  static timeUs_t lastTimeSentFPmAh = 0;
+
+  timeUs_t keepAlive = (currentTimeUs - lastTimeSentFPmAh)/1000;
```

Comment:
It's ms after dividing by 1000 (and counting using us would be probably better)

Reply:
Data type issue only, mS stored in a timeUs_t. OK, fixed it.

Reply:
Using us is IMO cleaner. And 2000 seconds is more than enough for this